core/cloudflow-operator/src/main/scala/cloudflow/operator/action/Action.scala

```diff
@@ -118,16 +129,24 @@ class CreateAction[T <: ObjectResource](
    */
   def execute(client: KubernetesClient)(implicit ec: ExecutionContext, lc: LoggingContext): Future[Action[T]] =
     for {
-      existing ← client.getOption[T](resource.name)(format, resourceDefinition, lc)
+      result <- executeCreate(client)
+    } yield new CreateAction(result, format, resourceDefinition, editor)
+
+  private def executeCreate(client: KubernetesClient)(implicit ec: ExecutionContext, lc: LoggingContext): Future[T] =
+    for {
+      existing ← client.getOption[T](resource.name)
       res ← existing
         .map { existingResource ⇒
           val resourceVersionUpdated =
             editor.updateMetadata(resource, resource.metadata.copy(resourceVersion = existingResource.metadata.resourceVersion))
-          client
-            .update(resourceVersionUpdated)(format, resourceDefinition, lc)
-            .map(o ⇒ new CreateAction(o, format, resourceDefinition, editor))
+          recoverFromConflict(
+            client
+              .update(resourceVersionUpdated),
```

Comment:
weird indentation. Maybe scalafmt doing weird things?

Reply:
Fixed it.